config/locales/links/en.yml

```
@@ -23,5 +23,5 @@ en:
     reverify: Please verify your identity again.
     sign_out: Sign out
     two_factor_authentication:
-      send_another_code: Send another code
+      try_again: Try again
```

We still need the "Send another code" text since it's used on the preceding screen.

```suggestion
      send_another_code: Send another code
      try_again: Try again
```